src/PlatformSockets.h

```diff
@@ -56,8 +51,6 @@ typedef socklen_t SOCKADDR_LEN;
 #define sockaddr_in6 sockaddr_in
 #define sin6_addr sin_addr
```

Comment:
do we still need these definitions for `sockaddr_in6` and similar fields? It would be nice to remove some `__vita__` ifdefs in the IPv6 code if possible (even if Vita never reaches the IPv6 codepaths).

Reply:
hm. i think we should define more network related structures into newlib.
it includes `netinet/tcp.h`, `sockaddr_in6` and `sockaddr_storage` (for enet).

I'll list up these and make the new pr if you allow this.
(or you can wait the next commits ;) )

Reply:
Let's do it in a separate PR. Try to see how many `__vita__` ifdefs are no longer necessary.